resilience4j-spring/src/main/java/io/github/resilience4j/circuitbreaker/configure/CircuitBreakerAspect.java

```diff
@@ -123,14 +127,16 @@ private Object defaultWebFlux(ProceedingJoinPoint proceedingJoinPoint, io.github
 			if (logger.isDebugEnabled()) {
 				logger.debug("Invocation of method '" + methodName + "' failed!", throwable);
 			}
-			throw throwable;
+
+			return recovery.apply(throwable);
```

Comment:
I think recovery should apply, when the Flux/Mono emits and error. We should use `onErrorResume`
Line 131 does not cover that.

Reply:
I fixed it.

> @lespinsideg Thx for your PR :) , i have some questions :
> The recovery type is forced to follow the recovery function type which has return type by default, now for the following cases :
> 
> * if the called logic is void ? how this will be optimally covered ?
> * for Reactive flow , in our case with Reactor types , the recovery logic most of the cases need to be properly handled by using for example onErrorResume , can u please add test cases for Spring web flux API with circuit breaker and see how it will be handled ?
> 
> Thx a lot in advance

Do you have any good idea for handling void return? In my opinion, `RecoveryFunction<Void>` could be used there.